src/main/java/frc/robot/commands/DefaultArmCommand.java

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+import frc.robot.constants.ArmConstants;
+import frc.robot.constants.JoystickConstants;
+import frc.robot.subsystems.ArmControlSubsystem;
+import frc.robot.subsystems.Input;
+
+public class DefaultArmCommand extends CommandBase {
+  
+  private final ArmControlSubsystem armControlSubsystem;
```

Comment:
This isn't the right convention. This should be mArmControlSubsystem". That is, member variables should start with a lowercase m to distinguish them from local variables and parameter names. And the constructor parameters below should not have a "tmp" prefix.

Reply:
Thanks for the feedback! I just updated all the files with the accurate conventions.